docker/nginx/nginx.conf

```diff
@@ -0,0 +1,82 @@
+worker_processes 4;
+
+events { worker_connections 1024; }
+
+http {
+	upstream app_server {
+        	server app-1:3000 fail_timeout=5s max_fails=5;
+	}
+
+	server {
+	    listen 80 default_server;
+	    listen [::]:80 default_server;
+	
+	    charset utf-8;
+	
+	    access_log /var/log/nginx/openhab.org-access.log;
+	    error_log /var/log/nginx/openhab.org-error.log;
+	    client_max_body_size 300m;
+	
+	    root /opt/openhabcloud;
+	
+	    location ~ /.well-known {
+	      allow all;
+	    }
+	
+		location /css {
```

Comment:
With a clean build of the docker images I get the following errors in the browser console:
Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/bootstrap/css/bootstrap.min.css".
(index):27 Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/js-plugin/pretty-photo/css/prettyPhoto.css".
(index):28 Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/js-plugin/rs-plugin/css/settings.css".
(index):32 Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/font-icons/custom-icons/css/custom-icons-ie7.css".
(index):31 Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/font-icons/custom-icons/css/custom-icons.css".
(index):29 Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/js-plugin/hoverdir/css/style.css".
(index):34 Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/css/layout.css".
(index):35 Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://192.168.178.27/css/colors.css".

And the page loads without styles. So I would suggest to change the config slightly and add:
include /etc/nginx/mime.types; to `http {` or add the mime types to each location block seperately, e.g. `add_header  Content-Type    text/css;` (which I would not prefer :))